demos/tests/args.py

```diff
@@ -52,6 +52,15 @@ def resolve(self, context):
         return str(context.dl_dir / context.model_info[self.name]["subdirectory"] / self.precision / (self.name + '.xml'))
 
 
+class ModelDirArg:
```

Comment:
```suggestion
class ModelFileArg:
```

The argument resolves to a file path, not a directory.

Reply:
yes, but the idea that this file is located in model downloading directory

Reply:
I get that, but the argument classes are named for what they are, not where they are.

Reply:
renamed